kodilla-spring/src/test/java/com/kodilla/spring/portfolio/BoardTestSuite.java

```diff
@@ -0,0 +1,35 @@
+package com.kodilla.spring.portfolio;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+public class BoardTestSuite {
+    @Test
+    public void testTaskAdd() {
+        //Given
+        ApplicationContext context = new AnnotationConfigApplicationContext(BoardConfig.class);
+        Board board = context.getBean(Board.class);
+
+        //When
+        board.getToDoList().addTask("Learn Project Patterns");
+        board.getInProgressList().addTask("Learning Spring");
+        board.getDoneList().addTask("Learned Stream");
+
+        String taskFromToDoList = board.getToDoList().getTasks().stream()
+                .findAny()
+                .toString();
```

Comment:
toString() na Optionalu to bardzo brzydki deasign. Zapisz 
        Optional<String> blabla = board.getToDoList().getTasks().stream()
                .findAny();
i pozniej
  assertTrue(blabla.isPresent());

Reply:
Przeczytaj o Optional'ach
http://www.baeldung.com/java-optional